platform/server/plugins/KibanaPluginValues.ts

```diff
@@ -23,8 +23,13 @@ export function createKibanaValuesForPlugin(
 ): KibanaPluginFeatures {
   return {
     logger: {
-      get: (...namespace) => {
-        return core.logger.get('plugins', pluginName, ...namespace);
+      get(namespace?: string) {
```

Comment:
Having default plugin context is very handy so I tried to keep it, but I hate this `join` in several places :/